### DISCUSSION TITLE
right align in table

### DISCUSSION THREAD

**Discussion by Andre1275** [Budibase/budibase#13875] [Help]
Hallo

How do I right-align a column with numbers in a query?
I am using the new table component.

My query 

 select logictyp, sum(price) as Summe from invoices where DATE_FORMAT(docdate, "%Y-%m") = concat("2024", "-", "03") and doctype = "RECHNUNG LIEFERANT" group by logictyp 

Thanks

Andre

**Comment by Andre1275** [comment]
It's done.

**Reply by AkinoYuki** [reply]
Hello,
How did you do? Can't find a way to center align the numbers in the columns of a Table Block component.
